app-emulation/distrobox/autobuild/build

```
@@ -0,0 +1,3 @@
+abinfo "Installing distrobox..."
+bash \
+    -C "$SRCDIR"/install --prefix "$PKGDIR"/usr
```

```suggestion
bash "$SRCDIR"/install --prefix "$PKGDIR"/usr
```

需要修正：`-C` 是用于编写补全支持的，不是这样让你调用脚本的。bash 后面可以直接跟脚本文件名